### DISCUSSION TITLE
Links within Tabfern itself.

### DISCUSSION THREAD

**Discussion by tscahill** [cxw42/TabFern#271] [General]
I'm uncertain if this is possible, so I'll just go ahead and ask. 

I'd like to be able to _copy links directly to Tabfern folders._ It is a great organizing tool, and I'd like to integrate more tightly into my daily routine. Right now that is with a hierarchical note taking app (Zim) that I manage my day-to-day. Links in my notes work fine for opening a particular url, but bypasses the tab grouping context. 

More than once I've found myself right-clicking on a tabfern folder hoping to get a link to the group.

**Comment by cxw42** [comment]
Interesting idea!  Could you tell me more about how you would use the links after you copied them?

Bug reports and feature requests are always welcome :) .

**Reply by tscahill** [reply]
OK. I have been using TabFern as one of the tools to organize the browser side of various projects I'm working on. The other relevant piece is the structured notes system I use in Zim. Those notes can and do have URL links, but clicking these will open a new instance of the page I've already got organized in TabFern. It would be nice to have a link to the Tabfern link to the anchor that will open (in the browser) the grouped version of the URL. It might be a little OCD of me to only want one instance of a page. 
The other use would be to click a link that opened TabFern and the particular folder I wanted to look at.
It is only in these odd-ball use-cases that these un-obvious needs arise. I'm not sure how much utility would arise from this proposed feature. All I can say for sure is if the links had been available to copy, I would have used them in my notes.

**Reply by cxw42** [reply]
> odd-ball use-cases

One of the great things about open-source software is the ability to consider those cases :) .

Tell me if I've got this right: you want to get a URL for a window or tab from TF and put it in Zim.  When you click the URL in Zim, you want it to:
* **Use case 1** be as if you had clicked that window or tab in TF
* **Use case 2** scroll the TF window to bring that window or tab into view

Yes?  No?

**Reply by tscahill** [reply]
Sorry for the long delay. Yes for Use case 1 and 2.
